crates/omnix-health/src/check/flake_enabled.rs

```diff
@@ -15,22 +15,156 @@ impl Checkable for FlakeEnabled {
         _: Option<&nix_rs::flake::url::FlakeUrl>,
     ) -> Vec<(&'static str, Check)> {
         let val = &nix_info.nix_config.experimental_features.value;
+
+        // Check if flakes are enabled either through config or DetSys installation
+        let flakes_enabled = match nix_info.nix_version.installation_type {
+            NixInstallationType::DeterminateSystems => {
+                // Determinate Systems Nix has flakes enabled by default
+                true
```

Comment:
Is it possible to disable flakes on DetSys installer? And if so, does the PR behave correctly?